app/grandchallenge/reader_studies/models.py

```diff
@@ -1091,11 +1091,15 @@ class Question(UUIDModel):
     class Meta:
         ordering = ("order", "created")
 
+    @property
+    def answer_type_display(self):
+        return self.get_answer_type_display()
+
```

Comment:
Added to allow use in template

Reply:
I'd rather keep to Django's API, you can call `{{ question.get_answer_type_display }}` from a django template, see their documentation about how template variables are resolved: https://docs.djangoproject.com/en/3.0/ref/templates/language/#variables